docs/demo/worker.js

```diff
@@ -121,22 +112,40 @@ function jsonString(input, level) {
 function loadVersion(ver) {
   let promise;
   if (versionCache[ver]) {
-    promise = Promise.resolve(versionCache[ver]);
+    promise = Promise.resolve();
   } else {
-    promise = fetch(ver)
-      .then(function(res) { return res.text(); })
-      .then(function(text) {
-        versionCache[ver] = text;
-        return text;
+    promise = import(ver + '/lib/marked.esm.js')
+      .catch(() => {
+        return fetch(ver + '/marked.min.js')
+          .then(function(res) { return res.text(); })
+          .then(function(text) {
+            try {
+              // eslint-disable-next-line no-new-func
+              Function(text)();
+            } catch (err) {
+              throw new Error('No esm or min build');
+            }
+            return (globalThis || global).marked;
+          });
+      })
+      .then((marked) => {
+        if (!marked) {
+          throw Error('No marked');
+        } else if (marked.marked) {
+          versionCache[ver] = marked.marked;
+        } else if (marked.default) {
+          versionCache[ver] = marked.default;
+        } else if (marked.parse) {
+          versionCache[ver] = marked;
+        } else {
+          throw new Error('Can not find marked');
```

Comment:
```suggestion
          throw new Error('Cannot find marked');
```

https://www.merriam-webster.com/grammar/cannot-vs-can-not-is-there-a-difference